### ISSUE TITLE
Sample on create everything programmatically

### ISSUE THREAD

**Issue by jslim89** [quemb/QMBParallaxScrollViewController#24]
I create everything programmatically on my project, when I try to plugin this plugin to my project, I encounter the issue that the base view cannot change to `UIImageView` and also cannot resize the `UIViewController`.

See http://stackoverflow.com/questions/25379152/change-base-view-of-uiviewcontroller-to-uiimageview

Can provide an sample project create everything programmatically?

**Comment by nrdgroove** [comment]
Did you got any solution for your issue?

**Comment by jslim89** [comment]
No. I end up using https://github.com/apping/APParallaxHeader this plugin
